### PR TITLE
refactor(clipboard): always use native event for pasting

### PR DESCRIPTION
**Pull request by wzhudev**
- [x] I am sure that the code is update-to-date with the `dev` branch.

Partial #1404

### CODE REVIEW

**Comment by wzhudev** [comment]
@yuhongz Could you please give a short explanation of what has been changed? So we can have @zhaolixin7 taking a look at this.

**Comment by yuhongz** [comment]
> @yuhongz Could you please give a short explanation of what has been changed? So we can have @zhaolixin7 taking a look at this.
@zhaolixin7 
1. When utilizing a Windows system, users can copy Excel content and paste it into a univ-sheet, preserving the font size information.
2. This feature is only effective when using keyboard shortcuts. If a user copies Excel content and uses the context menu for pasting, an error will be displayed, and the user will be guided to use keyboard shortcuts instead.
3. The behavior on Mac systems should remain unchanged.

File: packages/docs-ui/src/controllers/clipboard.controller.ts [hunk 1]
```diff
@@ -51,18 +51,16 @@ export class DocClipboardController extends RxDisposable {
     }
 
     private _initLegacyPasteCommand(): void {
-        if (!this._clipboardInterfaceService.supportClipboard) {
-            this._textSelectionRenderManager?.onPaste$.pipe(takeUntil(this.dispose$)).subscribe((config) => {
-                if (!whenDocOrEditor(this._contextService)) {
-                    return;
-                }
+        this._textSelectionRenderManager?.onPaste$.pipe(takeUntil(this.dispose$)).subscribe((config) => {
```
Comment (Jocs) [inline packages/docs-ui/src/controllers/clipboard.controller.ts:54]: 改用 native event 的原因是什么？

**Comment by zhaolixin7** [comment]
@yuhongz粘贴后字号与Excel不一致，路径： 复制Excel内容（字号为21）粘贴到内部字号显示为（14.5）

File: packages/docs-ui/src/controllers/clipboard.controller.ts (hunk 1 above)
Comment (wzhudev) [reply packages/docs-ui/src/controllers/clipboard.controller.ts:54]: 从 clipboardAPI 和 native event 读取的内容不一致，特别是在 windows 上，从 excel 里面复制的样式信息只在 native event 里面有

File: packages/sheets-ui/src/services/clipboard/clipboard.service.ts [hunk 2]
```diff
@@ -177,8 +180,16 @@ export class SheetClipboardService extends Disposable implements ISheetClipboard
                 : '';
 
         if (html) {
-            // Firstly see if the html content is in good format.
-            // In another word, if it is copied from any spreadsheet apps (including Univer itself).
+            // Firstly see if the html content is from Excel
+            if (this._platformService.isWindows && (await clipboardItemIsFromExcel(html))) {
+                this._notificationService.show({
+                    type: 'warning',
+                    title: '粘贴提示',
```
Comment (wzhudev) [inline packages/sheets-ui/src/services/clipboard/clipboard.service.ts:187]: Need i18n here.

Comment (wzhudev) [inline packages/sheets-ui/src/services/clipboard/clipboard.service.ts:187]:
```suggestion
                    title: '请使用快捷键粘贴',
```

File: packages/sheets-ui/src/services/clipboard/clipboard.service.ts [hunk 3]
```diff
@@ -177,8 +180,16 @@ export class SheetClipboardService extends Disposable implements ISheetClipboard
...
+                    content: '无法粘贴内容，请试试快捷键 Ctrl+V 。',
```
Comment (wzhudev) [inline packages/sheets-ui/src/services/clipboard/clipboard.service.ts:188]:
```suggestion
                    content: '检测到 Excel 内容，请使用快捷键粘贴',
```

**Pull request merged**